src/widgets/tdarr/WorkersPanel.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  Center,
+  Group,
+  Progress,
+  ScrollArea,
+  Stack,
+  Table,
+  Text,
+  Title,
+  Tooltip,
+} from '@mantine/core';
+import { IconHeartbeat, IconTransform } from '@tabler/icons-react';
+import { useTranslation } from 'next-i18next';
+import { TdarrWorker } from '~/server/api/routers/tdarr';
+import { WidgetLoading } from '~/widgets/loading';
+import { Filename } from '~/widgets/tdarr/Filename';
+
+interface WorkersPanelProps {
+  workers: TdarrWorker[] | undefined;
+  isLoading: boolean;
+}
+
+export function WorkersPanel(props: WorkersPanelProps) {
+  const { workers, isLoading } = props;
+
+  const { t } = useTranslation('modules/tdarr-queue');
+
+  if (isLoading) {
+    return (
+      <Stack
+        justify="center"
+        style={{
+          flex: 1,
+        }}
+      >
+        <WidgetLoading />
+      </Stack>
+    );
+  }
+
+  if (!workers?.length) {
+    return (
+      <Center
+        style={{ display: 'flex', alignItems: 'center', justifyContent: 'center', flex: '1' }}
+      >
+        <Title order={3}>{t('views.workers.table.empty')}</Title>
+      </Center>
+    );
+  }
+
+  return (
+    <ScrollArea style={{ flex: '1' }}>
+      <Table style={{ tableLayout: 'fixed' }}>
+        <thead>
+          <tr>
+            <th>{t('views.workers.table.header.name')}</th>
+            <th style={{ width: 60 }}>{t('views.workers.table.header.eta')}</th>
+            <th style={{ width: 175 }}>{t('views.workers.table.header.progress')}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {workers.map((worker) => (
+            <tr key={worker.id}>
+              <td>
+                <Group spacing="xs" noWrap>
+                  <div>
```

Comment:
With: 
![image](https://github.com/ajnart/homarr/assets/31685172/4ce51229-df52-498b-b3b9-c36118d6603b)
Without: 
![image](https://github.com/ajnart/homarr/assets/31685172/18afbff7-4fc8-4c75-9de4-e237e1fc8673)

Reply:
That's strange, it shouldn't be doing that since you specified a size for the icon.
It's actually working fine for me in both chromium and firefox, what browser base are you using?
In any case it's fine, I was just curious.